src/Resources/views/formatter/badge.html.twig

```diff
@@ -0,0 +1,20 @@
+{% set classes = '' %}
+{% if background_color_hex is defined and background_color_hex is not null %}
+    {% set classes = classes ~ ' bg-[' ~ background_color_hex ~ ']' %}
+{% endif %}
+
+{% if background_color_class is defined and background_color_class is not null %}
+    {% set classes = classes ~ ' ' ~ background_color_class %}
```

Comment:
Nit: Twig supports [String Interpolation](https://twig.symfony.com/doc/3.x/templates.html#string-interpolation). So, this can be written in an easier readable fashion:

```twig
{% set classes = "#{classes} #{background_color_class}" %}
```